html-report-creator.js

```diff
@@ -92,8 +92,12 @@ module.exports = (result) => {
       });
     }
   });
-  writeFile(process.env.JEST_TEST_REPORT_PATH || path.join(process.cwd(), 'test-results/test-report.html'), htmlOutput);
+  writeFile(process.env.JEST_TEST_REPORT_PATH || path.join(process.cwd(), 'test-results/test-report.html'), htmlOutput.end({
+    pretty: true,
+    indent: '  ',
+    newline: '\n',
+    allowEmpty: false,
+  }));
```

Comment:
given the length of the line, now that there's a function call with options maybe splitting the arguments into lines would be nicer? not a blocker
```suggestion
  writeFile(
    process.env.JEST_TEST_REPORT_PATH || path.join(process.cwd(), 'test-results/test-report.html'),
    htmlOutput.end({
      pretty: true,
      indent: '  ',
      newline: '\n',
      allowEmpty: false,
    })
  );
```

Reply:
Taken care of in 008ecd9